packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -4226,7 +4258,7 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
       gotoValueSelectedFilter,
       partitionConfig,
     } = this.state;
-    if (!isReady) {
+    if (!isReady || partitionConfig?.mode === 'loading') {
```

Comment:
I'm going to need to think about this a bit more - we should be showing the loading spinner while it's still loading.